File: ASSIGNMENT 3/number of digits ina number.cpp
#include <iostream>
using namespace std;
class Number
 {
    int num, temp;
    int count = 0;
    public: void Check_number()
    {
    cout << "Enter any number : ";
    cin >> num;
    temp = num;
    while(temp != 0)
        {
            count++;
            temp /= 10;
        }

    cout << endl << "Total digits in " << num << " : "  << count;


}
};
int main()
{
    Number n1;
    n1.Check_number();
}

File: ASSIGNMENT 4(java)/numberofdigitsinanumber.java
import java.util.Scanner;
class Number
{
   public static void main(String args[])
{
      int num,temp;
      int count=0;
      System.out.println("Enter the number:");
      Scanner sc = new Scanner(System.in);
      num = sc.nextInt();
      temp=num;
      while(temp!=0)
      {
        count++;
        temp/=10;
      }
      System.out.println("Total digits in: "+num+":"+count);
   }
  
   }
